runtime/pom.xml

```diff
@@ -44,36 +44,43 @@
     <finalName>runtime</finalName>
     <plugins>
       <plugin>
-        <groupId>org.wildfly.swarm</groupId>
-        <artifactId>wildfly-swarm-plugin</artifactId>
-        <version>${wildfly.swarm.version}</version>
-        <configuration>
-          <mainClass>com.redhat.ipaas.runtime.Main</mainClass>
-          <jvmArguments>
-            <jvmArgument>-server</jvmArgument>
-            <jvmArgument>-Xms256m</jvmArgument>
-            <jvmArgument>-Xmx512m</jvmArgument>
-          </jvmArguments>
```

Comment:
No engineers are going to be hurt in the development of ipaas... well unless we get on and merge this quickly anyway...

But I can't be bothered to argue about this really - we've got functionality to add and this is an important change - so let's go with the arbitrary numbers.